### PR TITLE
print warnings when tx body is not correct for tests

### PR DESCRIPTION
**Pull request by mkv-vcm**

### CODE REVIEW

File: test/integration/signTx.test.ts [hunk 1]
```diff
@@ -15,80 +15,46 @@ import {
 
 // ========================================   BYRON   ========================================
```
Comment (janmazak) [inline test/integration/signTx.test.ts:16]: Remove?

File: test/test_utils.ts [hunk 2]
```diff
@@ -70,3 +73,11 @@ export const Networks = {
         protocolMagic: 47,
     },
 }
+
+type TxHash = FixlenHexString<32>
+
+export function hashTxBody(txBody: string): TxHash {
```
Comment (janmazak) [inline test/test_utils.ts:79]:
```suggestion
export function hashTxBody(txBodyHex: string): TxHash {
```

**Review by janmazak — APPROVED** [review]
I presume the negative tests are refactored elsewhere.

File: test/integration/signTx.test.ts (hunk 1 above)
Comment (mkv-vcm) [reply test/integration/signTx.test.ts:16]: Yeah.

File: test/integration/__fixtures__/signTx.ts [hunk 3]
```diff
@@ -532,6 +532,7 @@ export type TestcaseByron = {
   testname: string
   tx: Transaction
   signingMode: TransactionSigningMode
+  txBody?: string,
```
Comment (janmazak) [inline test/integration/__fixtures__/signTx.ts:535]:
```suggestion
  txBody?: string
```

**Review by janmazak — APPROVED** [review]

**Pull request merged**